tests/utils.go

```diff
@@ -3373,6 +3374,17 @@ func BeforeAll(fn func()) {
 	})
 }
 
+func SkipIfNoWindowsSysprepImage(virtClient kubecli.KubevirtClient) {
```

Comment:
@borod108 We would like to reduce duplicate code. Can you please create a generic function which will accept 3 parameters: the virt client, image name and skip message?

Reply:
Good point!
Changed to a func with 2 params, the skip message does not really have to be custom beyond the name of the missing image. hope it is ok.